### PR TITLE
bach 0.0.10 (new formula)

### PR DESCRIPTION
**Pull request by fabiob**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by fabiob** [comment]
@compose @codepope

File: Formula/bach.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Bach < Formula
+  desc "Command-line for the Compose API"
+  homepage "https://github.com/compose/bach"
+  url "https://github.com/compose/bach/releases/download/0.0.10/bach_Darwin_x86_64.tar.gz"
```
Comment (dunn) [inline Formula/bach.rb]: This needs to be compiled from source.

Comment (fabiob) [inline Formula/bach.rb]: I've updated the formula so it builds from source.

File: Formula/bach.rb [hunk 2]
```diff
@@ -0,0 +1,108 @@
+require "language/go"
+
+class Bach < Formula
+  desc "Command-line for the Compose API"
+  homepage "https://github.com/compose/bach"
+  url "https://github.com/compose/bach.git", :tag => "0.0.10"
```
Comment (dunn) [inline Formula/bach.rb]: If there aren't submodules or anything requiring a clone, we prefer the tarball: https://github.com/compose/bach/archive/0.0.10.tar.gz

If we do need to fetch it with git, add a `revision` as well so we can catch if it's re-tagged.

**Comment by JCount** [comment]
@fabiob Any progress with this?

File: Formula/bach.rb [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+      :revision => "210eee5cf7323015d097341bcf7166130d001cd8"
+  end
+
+  go_resource "gopkg.in/yaml.v2" do
+    url "https://gopkg.in/yaml.v2.git",
+      :revision => "cd8b52f8269e0feb286dfeef29f8fe4d5b397e0b"
+  end
+
+  def install
+    rm_rf "vendor" # fix for https://github.com/compose/bach/issues/13
```
Comment (ilovezfs) [inline Formula/bach.rb]: We'll need to wait for a new release that has a fully up to date vendor directory.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

**Comment by ilovezfs** [comment]
@fabiob there is now a [0.1.0 release](https://github.com/compose/bach/releases/tag/0.1.0) if you'd like to refresh this!

**Comment by ilovezfs** [comment]
@fabiob thanks for opening this. I'm going to close it for now since it's for an outdated version, but please feel free to open a new PR for 0.1.0 if you'd still like to see this added to Homebrew.

**Pull request closed without merging**